src/solvers/refinement/README.md

```diff
@@ -6,6 +6,22 @@
 
 \section string_solver_interface String solver interface
 
+After the input program of CProver has been converted to a goto-program and then
+to a set of equations, these equations are fed to a solver.
+For program verification, this is generally the last step in CProver, but we can
+for instance ask for a trace to be generated from the result of the solver.
+
+The basic role of the solver is to answer whether the set of equations given
+is satisfiable.
+In general, this corresponds to the possibility of violating an assertion of the
+input program.
```

Comment:
Maybe instead of "in general" say "In bounded model checking"?

Reply:
What I meant is that, it could be something else than an assertion, like null pointer dereference and out of bound access.

Reply:
Oh, then maybe the present wording is too brief. As I you can see, I have read something very different into those words.